ONSdigital/eq-author
# 89 Harden storybook

### What is the context of this PR?

Improves reliability of React Storybook system:

1. Upgraded to latest version
2. Added some stories for existing components
3. Added add-ons
4. Added smoke-test to check all pages are navigable
5. Deploys storybook to gh-pages on successful merge of a PR

### How to review 

- `yarn storybook`
- Ensure app runs, pages are navigable
- `yarn smoke-test`
- Ensure smoke-tests pass
- `yarn build`
- Ensure build completes successfully
